src/DisplayEditableData.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+function display(object $card)
```

Comment:
Using object as a type hint should be avoided, it's too vague. Instead you should use your PlayerCard class

Reply:
Also change the name of this function, display is too vague